apps/broadway_configurator/lib/broadway_configurator.ex

```diff
@@ -1,44 +0,0 @@
-defmodule BroadwayConfigurator do
```

Comment:
I think Jeff mentioned this earlier; I'd also be curious as to why we no longer need the broadway_configurator.

Reply:
I don’t even see broadway period

Reply:
^ This. :)

Reply:
What I mean is, after these deletions I don’t see where we’re starting broadway anymore which means we are either not doing gen_stage based consumption or we’re planning to roll our own.